apps/web/pages/api/auth/signup.ts

```diff
@@ -17,7 +18,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   const data = req.body;
-  const { email, password } = data;
+  const { email, password, language } = data;
```

Comment:
nit: Might be a good idea to introduce zod schema for this.

Reply:
I've actually done this in my signup migration branch - i can bring it over for this PR if youd like? :P